spec/descriptions/updateGroup.md

```diff
@@ -0,0 +1,7 @@
+Add a permission to a group. Permissions are strings associated with the group that some resources requires to fulfill requests.
+
+Some examples of `Permissions`:
+
+- `CAN_VIEW_AUDIT_LOG`
+- `ACCESS_WEBSITES` (only enforced when permissions also contains `RESTRICTED_ACCESS`)
+- More...
```

Comment:
Well, there are many more permissions. I did not wanted to name them all here because more may be added.I wil name them all and add there may be more.